include/pinocchio/spatial/inertia.hpp

```diff
@@ -889,6 +960,366 @@ namespace pinocchio
 
   }; // class InertiaTpl
 
+  /**
+   * @brief A structure representing a pseudo inertia matrix.
+   *
+   * References:
+   * - Wensing, Patrick M., Sangbae Kim, and Jean-Jacques E. Slotine. "Linear matrix inequalities
+   * for physically consistent inertial parameter identification: A statistical perspective on the
+   * mass distribution." IEEE Robotics and Automation Letters 3.1 (2017): 60-67.
+   */
+  template<typename Scalar, int Options>
+  struct PseudoInertiaTpl
+  {
+    typedef Eigen::Matrix<Scalar, 4, 4, Options> Matrix4;
+    typedef Eigen::Matrix<Scalar, 3, 1, Options> Vector3;
+    typedef Eigen::Matrix<Scalar, 3, 3, Options> Matrix3;
+    typedef Eigen::Matrix<Scalar, 10, 1, Options> Vector10;
+    typedef LogCholeskyParametersTpl<Scalar, Options> LogCholeskyParameters;
+
+    Scalar mass;    ///< Mass of the pseudo inertia
+    Vector3 h;      ///< Vector part of the pseudo inertia
+    Matrix3 Sigma;  ///< 3x3 matrix part of the pseudo inertia
```

Comment:
```suggestion
    Matrix3 sigma;  ///< 3x3 matrix part of the pseudo inertia
```

member should be in camelCase.